qiling/os/windows/dlls/kernel32/winnls.py

```diff
@@ -158,3 +158,25 @@ def hook_LCMapStringA(ql, address, params):
 })
 def hook_LCMapStringEx(ql, address, params):
     return _LCMapString(ql, address, params)
+
+
+# LANGID GetUserDefaultUILanguage();
+@winapi(cc=STDCALL, params={
+})
+def hook_GetUserDefaultUILanguage(ql, address, params):
+    # TODO find better documentation
+    # https://docs.microsoft.com/it-it/windows/win32/intl/language-identifiers
+    ql.dprint(0, "[!] Sample is checking user language!")
+    lang = 0x445
+    return lang
+
+
+# LANGID GetSystemDefaultUILanguage();
+@winapi(cc=STDCALL, params={
+})
+def hook_GetSystemDefaultUILanguage(ql, address, params):
+    # TODO find better documentation
+    # https://docs.microsoft.com/it-it/windows/win32/intl/language-identifiers
+    ql.dprint(0, "[!] Sample is checking system language!")
```

Comment:
my suggestion is you might want to choose different symbols for this dprint, example [$] or [=] just a example. the reason is

1. easier to capture for malware analysis. 
2. if we gonna make a sandbox based on qiling, we just need to capture a specific symbols
3.  [!] we keep it for error, or raise

Reply:
Agree on the new symbol, will change it.
Agree on the level, forgot to change it

Reply:
f4cf13f should resolve the problem